### DISCUSSION TITLE
How to always get a challenge?

### DISCUSSION THREAD

**Discussion by TestDeTuring** [FlareSolverr/FlareSolverr#805] [Q&A]
Hello, is there any way to always get a challenge? because i need the cf_clearance cookies in order for my program to work, i saw that you could change the UA for that, but you can't do it from flare, is there any way to change the UA if i fork it? if so, how?

**Comment by ngosang** [comment]
I don't understand your question. You can't change the FlareSolverr User-Agent. If you want a site to test FlareSolverr try https://nowsecure.nl
